scripts/coinweight.lua

```diff
@@ -184,7 +188,7 @@ function onInit()
 	else
 		Debug.chat("ruleset has no denominations defined in Coins Weight. If submitting denominations for inclusion, tell bmos ruleset name is: " .. sRuleset)
 	end
-	
+
```

Comment:
Whitespace (luacheck).

Reply:
Notepad++ or GitHub (I forget which) automatically adds this, so it's just going to come back. But sure.